test/integration/multitenant_enforcement_test.rb

```diff
@@ -14,6 +14,7 @@ class MultitenantEnforcementTest < ActionDispatch::IntegrationTest
     plan.update_column(:tenant_id, @provider.tenant_id + 1)
     get admin_service_application_plans_path(service)
   rescue => exception
+    assert_predicate exception, :cause
```

Comment:
Why is this needed?

Reply:
if `cause` is `nil`, then we don't have any indication in error output what was `exception`. With this assertion, we will see at least what was `exception`.